README.md

```diff
@@ -177,8 +177,8 @@ overwritten during the deployment.
 
 4. Execute provisioning script on your host PC and pass obtained IP address as parameter:
 
-   ```sh
-   aos-prov provision -u 192.168.10.124
+   ```console
+   user@host:~$ aos-prov provision -u 192.168.10.124
```

Comment:
ditto

and remove next empty line

Reply:
This is the example I meant above: you couldnt "just copy the whole block and past into your terminal. I will reformat it according to the requirement above.
Also the empty line was not introduced by this patch, it was here before because it is a part of the output.